### DISCUSSION TITLE
How do I get the PairExtraordinate Badge?

### DISCUSSION THREAD

**Discussion by yeslee-v** [community/community#20684] [Discussions] [answered]
I'm already coauthor in some repositories, but didn't get that badge 🤣 

Is it error or I know wrong?

**Comment by aaronlamz** [comment, marked as answer]
You need to attribute a commit to more than one author by adding one or more Co-authored-by trailers to the commit's message. Co-authored commits are visible on GitHub.
More details here:
https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

**Comment by ahmadee21** [comment]
how can i start

**Comment by ahmadee21** [comment]
i need to learn how to earn thats why i joined please who can teach me how

**Comment by sheilalarmour** [comment]
Play Minecraft!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by sheilalarmour** [comment]
Sleep!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
